CONTRIBUTING.md

```diff
@@ -2,22 +2,29 @@
 
 This doc is intended for contributors to Go SDK (hopefully that's you!)
 
-**Note:** All contributors also need to fill out the [Temporal Contributor License Agreement](https://gist.github.com/samarabbas/7dcd41eb1d847e12263cc961ccfdb197) before we can merge in any of your changes
+**Note:** All contributors also need to fill out the [Temporal Contributor License Agreement](https://gist.github.com/samarabbas/7dcd41eb1d847e12263cc961ccfdb197) before we can merge in any of your changes.
 
 ## Development Environment
 
 * [Go Lang](https://golang.org/) (minimum version required is 1.14):
-  - Install on OS X with `brew install go`.
-  - Install on Ubuntu with `sudo apt install golang`.
+  - Ubuntu: `sudo apt install golang`.
+  - OS X: `brew install go` and add this to your `.bashrc`:
+
+```
+export GOPATH=$HOME/go
+export GOROOT="$(brew --prefix go)/libexec"
+export PATH="$PATH:${GOPATH}/bin:${GOROOT}/bin"
+```
```

Comment:
Code code block is not properly nested under the bullet (see https://gist.github.com/clintel/1155906)